idr/attributes.py

```diff
@@ -109,21 +113,19 @@ def annotation_ids_by_field(conn,
     return unwrap(conn.getQueryService().projection(q, params))[0]
 
 
-def get_phenotypes_for_gene(gene_name, screenid=None):
+def get_phenotypes_for_gene(idr_base_url, session, gene_name, screenid=None):
```

Comment:
`idr_base_url` and `session` seem to be required by several methods. Have you thought about refactoring this into a class?

Reply:
That would be a good idea, I will first work on the raw basics in this PR and do the refactoring bits in a separate PR if that sounds good.